chapters/001-getting-started/README.md

```diff
@@ -236,17 +236,17 @@ After seeing the ARP packets go back and forth (which establish the ability for
 
 At this point, there are a whole bunch of manual steps to get all this going.  Now that we have proven to ourselves that we know how to do this all manually, let's automate it! We have more containers to bring up and networks to build, and doing that all by hand will be a lot of toil.
 
-We are going to use the `docker compose` command which uses the [docker-compose.yml](docker-compose.yml) file in this directory to build, configure, and start our two containers on our network.
+We are going to use the `docker-compose` command which uses the [docker-compose.yml](docker-compose.yml) file in this directory to build, configure, and start our two containers on our network.
```

Comment:
from [this stackoverflow answer](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)

> The docker compose (with a space) is a newer project to migrate compose to Go with the rest of the docker project. This is the v2 branch of the [docker/compose](https://github.com/docker/compose/tree/v2) repo. It's been first introduced to Docker Desktop users, so docker users on Linux didn't see the command. In addition to migrating to Go, it uses the compose-spec, and part of the rewrite may result in behavior differences.

> The original python project, called docker-compose, aka v1 of docker/compose repo, has now been deprecated and development has moved over to v2. To install the v2 docker compose as a CLI plugin on Linux, supported distribution can now install the docker-compose-plugin package. E.g. on debian, I run apt-get install docker-compose-plugin.

So I don't think this change is a good idea? We could have a caveat that if you're using `docker-compose` you should upgrade your shit?

Reply:
warble garble. ok. lemme roll back the last commit and let's talk about how we want to approach this on Wednesday.